api/Api/AssetManager/AssetManager.cs

```diff
@@ -360,6 +360,22 @@ private Inner()
                            throw new Exception("Import of image/sprite asset: wrong number of arguments");
                        return new SpriteAsset(name, perm, sourcePath);
                    });
+
+                RegisterAssetType("image/texture", new[] {".png", ".jpeg"},
```

Comment:
Both SpriteAsset and TextureAsset are registered to use these file extensions, so you will have to remove one or the other or there may be problems with how it automatically deduces which type of asset to import a file as.

I made SpriteAsset a while ago to be a part of the UI stuff, but it seems like it is not used. Should we delete it? Or do you think it might be useful for anything.

Reply:
Im gonna actually get rid of TextureAsset just because a sprite is a texture with a little more information and we can just grab the texture from a sprite.